README.rst

```diff
@@ -22,171 +21,3 @@ __ https://openedx.atlassian.net/wiki/spaces/COMM/pages/3185508485/BTR+Meeting+M
 More details about the working group are at the `BTR page in the wiki`__.
```

Comment:
Some people might be used to finding the roles & assignments in this readme, so I recommend adding some text to help them find the new home for those things.
```suggestion
Roles, role assignments, and more details about the working group are at the `BTR page in the wiki`__.
```